src/Passes/examples/RandomWalkPE2/QSharpVersion/Example.qs.orig

```diff
@@ -0,0 +1,146 @@
+<<<<<<< HEAD
```

Comment:
Was this file committed accidentally? It looks like it is not in use, given that it still has the diff markers in it. Also, there is a bug fix to this particular example (time and theta were accidentally swapped).